### PR TITLE
(maint) re-add acceptance tests, lite-engine can run in HTTP mode

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -28,33 +28,111 @@ volumes:
...
+    commands:
+      - go build -o lite-engine
+  - name: server
+    detach: true
+    commands:
+      - touch .env
+      - echo "SERVER_HTTPS_BIND=localhost:9000" >> .env
+      - echo "CLIENT_HTTPS_BIND=localhost:9000" >> .env
+      - echo "SERVER_INSECURE=true" >> .env
+      - echo "SERVER_INSECURE=true" >> .env
```
Comment (shubham149) [inline .drone.yml:51]: Remove duplicate line

File: go.mod [hunk 2]
```diff
@@ -7,8 +7,8 @@ replace github.com/docker/docker => github.com/docker/engine v17.12.0-ce-rc1.0.2
 require (
 	github.com/cenkalti/backoff/v4 v4.1.1
 	github.com/docker/distribution v2.8.0+incompatible
-	github.com/docker/docker v0.0.1
-	github.com/docker/go-connections v0.4.0
+	github.com/docker/docker v1.3.3
+	github.com/docker/go-connections v0.4.0 // this is fake as we are using github.com/docker/engine, this makes the security warning go away
```
Comment (shubham149) [inline go.mod:11]: Move the comment to L10

File: server/server.go [hunk 3]
```diff
@@ -23,24 +24,33 @@ type Server struct {
...
-	tlsOptions := tlsconfig.Options{
-		CAFile:             s.CAFile,
-		CertFile:           s.CertFile,
-		KeyFile:            s.KeyFile,
-		ExclusiveRootPools: true,
+	var tlsConfig *tls.Config
+	if s.Insecure {
+		tlsConfig = nil
+		logrus.Warnln("RUNNING IN INSECURE MODE")
+		logrus.Warnln("RUNNING IN INSECURE MODE")
```
Comment (shubham149) [inline server/server.go:36]: Remove duplicate log lines

Comment (tphoney) [reply server/server.go:36]: i would prefer to leave it in, as i dont want an end user to accidentally miss that they are running in insecure mode.

**Review by eoinmcafee00 — APPROVED** [review]

**Pull request merged**